packages/cozy-notifications/src/webpack/config.js

```diff
@@ -4,9 +4,12 @@ import path from 'path'
 export default {
   resolve: {
     alias: {
-      handlebars: 'handlebars/runtime.js'
+      handlebars: 'handlebars/dist/handlebars.min.js'
     }
   },
+  optimization: {
+    minimize: false
```

Comment:
I'd be curious to see the bundles generated with this boolean as it disables minification for js files. The problem isn't that all our projects don't have [TerserWebpackPlugin](https://webpack.js.org/plugins/terser-webpack-plugin/) as a dependency

Reply:
This is something we already encountered with @Merkur39 when working on banks. I remember it was something like this stuff, but I don't remember how we have fixed it.

Reply:
> The problem isn't that all our projects don't have TerserWebpackPlugin as a dependency

Non, logiquement toutes nos app ont ce plugin grâce à cozy-scripts (bon c'est pas une dep directe de cozy-script, mais ça vient avec webpack blablabla. On pourrait corriger ça). 

Quand une app à une dep directe à ce plugin c'est à cause de storybook qui fait qu'on a des versions incompatibles, donc on force la bonne version de terser (genre https://github.com/cozy/cozy-home/commit/b0c5915) 

@zatteo on a retrouvé avec @Merkur39 ce qu'on avait fait : 
- Alors on n'était pas passé par la config webpack de cozy-notif (je sais plus pourquoi) mais on avait bien modifié l'alias https://github.com/cozy/mespapiers/pull/336/commits/8021643c34c033bf53d6c8b674175c54afd89654 . Par contre, y a un peu plus de config supplémentaire apparamment
- Et on avait aussi fait la minimization à false https://github.com/cozy/mespapiers/commit/4b2ad1e98cd9a091bd80741d0b04d075be1dd4be

Reply:
Ah bah en fait @zatteo y a pas de diff de config, j'avais pas vu tout ça https://github.com/cozy/cozy-libs/blob/ff9df5c87b7daf170a55c5f7cfc9502c6b65ad47/packages/cozy-notifications/src/webpack/config.js#L17-L18